mongorestore/main/mongorestore.go

```diff
@@ -47,8 +47,8 @@ func main() {
 	if err = restore.Restore(); err != nil {
 		log.Logvf(log.Always, "Failed: %v", err)
 		if err == util.ErrTerminated {
```

Comment:
this conditional isn't needed anymore. it can be replaced with `os.Exit(ExitFailure)`

Reply:
fixed